server/controllers/feeds/feeds.coffee

```diff
@@ -0,0 +1,27 @@
+CONFIG = require 'config'
+__ = require('config').universalPath
+_ = __.require 'builders', 'utils'
+root = CONFIG.fullPublicHost()
+items_ = __.require 'controllers', 'items/lib/items'
+error_ = __.require 'lib', 'error/error'
+RSS = require 'rss'
+limit = 100
+
+module.exports =
+  get: (req, res, next)->
+    feed = new RSS({"title":"New Public Items"})
+    { query } = req
+    items_.publicByDate limit
```

Comment:
better start with user-specific public inventories than with the global public inventory, which sole interest is to display examples on [/welcome](http://inventaire.io/welcome)